etm/ess-interface-spec/etm-ess-api.yaml

```diff
@@ -2086,6 +2088,197 @@ components:
+        negotiated_operators:
+          $ref: '#/components/schemas/NegotiatedOperators'
+        regular_updates:
+          $ref: '#/components/schemas/RegularUpdate'
+        trajectory:
+          $ref: '#/components/schemas/Trajectory'
+    AircraftRegistration:
+      type: string
+      pattern: "[A-Z0-9]{1,7}"
+      description: |
+        A unique, alphanumeric string that identifies a civil aircraft and consists of the Aircraft
+        Nationality or Common Mark and an additional alphanumeric string, the registration mark,
+        assigned by the state of registry or common mark registering authority. [FIXM]
+      minLength: 1
+      maxLength: 7
+      externalDocs:
+        description: See FIXM 4.2.0 for further information.
+        url: https://fixm.aero/releases/FIXM-4.2.0/doc/schema_documentation/Fixm_AircraftRegistrationType.html#LinkBC
+    Operator:
+      type: object
+      description: >-
+        The information about an operator, identifier, email, telephone number
+      properties:
+        operator_name:
+          $ref: '#/components/schemas/OperatorName'
+        email:
+          type: string
+          format: email
+        phone_number:
+          type: string
+    OperatorName:
+      description: The identifier of the ETM Operator.
+      type: string
+      pattern: "[A-Z]{3}"
+      example: "AAL" 
+      externalDocs:
+        description: See FIXM 4.2.0 for further information.
+        url: https://www.fixm.aero/releases/FIXM-4.2.0/doc/schema_documentation/Fixm_AircraftOperatorType.html#Link1E3
+    MissionType:
+      type: string
+      description: >-
+        Designation of mission type, whether public or private. Default to public
+      enum:
+      - PUBLIC
+      - PRIVATE
+      default: PUBLIC
+    MissionDescription:
+      type: string
+      description: >-
+        Description of the mission, e.g. describing the mission, such as fly to the Gulf of 
+        Mexico and loiter for 6 months providing a service @spencer this should be a enum/type
+        field
+      maximum: 1000
+    ContingencyHandlingDescription:
+      type: string
+      description: >-
+        Description of the contigency handling procedures @spencer this should be a enum/type
+        field
+      maximum: 1000
+    CommonOperatingProcedure:
+      type: integer
+      description: common operating procedure, integer values denote a specific type of rules
+        by which the operation will interact with other operations by default. Does not preclude
+        operator-operator negotiation or agreements @todo better description @spencer
+      default: 1
+    NegotiatedOperators:
+      type: array
+      description: >-
+        List of other operators that the operator of this operation has agreements with for the
+        purposes of negotiation outside of the common operating procedures.
+      items:
+        $ref: '#/components/schemas/OperatorName'
```

Comment:
This is supposed to be an array (max length of 1024) of free-format strings (of max length 256).